com.eclipsesource.json/pom.xml

```diff
@@ -21,8 +21,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.2</version>
         <configuration>
-          <source>1.5</source>
-          <target>1.5</target>
+          <source>1.8</source>
+          <target>1.8</target>
```

Comment:
These two lines are needed to solve the problem found [here on the discussion](https://github.com/ralfstx/minimal-json/issues/94#issuecomment-381354085) of issue #94